serverless-logic/serverless-logic-managing-services/serverless-logic-configuring-openAPI-services-endpoints.adoc

```diff
@@ -0,0 +1,101 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-logic-configuring-openAPI-services-endpoint"]
+= Configuring OpenAPI services endpoints
+:context: serverless-logic-configuring-openAPI-services-endpoints
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+{ServerlessLogicProductName} uses the `kogito.sw.operationIdStrategy` property to generate the REST client for invoking services defined in OpenAPI documents. This property determines how the configuration key is derived for the REST client configuration.
```

Comment:
Everything from here to line 98 seems like it should be in a REFERENCE module rather than an assembly. 

Consider writing an introductory statement under the `toc::[]` and moving the rest of this to a new assembly module describing the function of the openAPI services endpoints.